test/scm/svn_test.go

```
@@ -0,0 +1,257 @@
+	name = fmt.Sprintf("branches/test-%d", time.Now().UnixMilli())
+	return
+}
+
+// deleteSvnBranch removes a branch path from the remote for cleanup.
+func deleteSvnBranch(r scm.SCM, branch string) (err error) {
+	svn, cast := r.(*scm.Subversion)
+	if !cast {
+		return
+	}
+	identity := svn.Remote.Identity
+	cmd := osexec.Command(
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)